py-polars/pypolars/frame.py

```diff
@@ -100,6 +100,28 @@ def read_csv(
         n_threads: Optional[int] = None,
         dtype: "Optional[Dict[str, DataType]]" = None,
     ) -> "DataFrame":
+        """
+        Read a comma-seperated value file into a Dataframe.
+
+        Parameters
+        ---
+        - file `str`, `TextIO`  
```

Comment:
Types shouldn't be defined in the docstring. They are defined already in the arguments. This ensures a single source of truth.

Reply:
The same counts for the defaults values.